bucket/python.json

```diff
@@ -28,6 +45,19 @@
             "}"
         ]
     },
+    "pre_install": [
+        "'install-pep-514.reg', 'uninstall-pep-514.reg' | ForEach-Object {",
+        "    $py_root = \"$dir\".Replace('\\', '\\\\')",
+        "    $py_version = [regex]::match($version,'(\\d+.\\d+)').Groups[0].Value",
```

Comment:
You need only `3.8` for example?

Reply:
Yes. py_version needs to be only major.minor. If it includes micro, then it doesn't work. Is there a smarter way to get that?